src/client/Microsoft.Identity.Client/json/Utilities/ReflectionUtils.cs

```diff
@@ -41,7 +41,7 @@
 
 namespace Microsoft.Identity.Json.Utilities
 {
-#if (DOTNET || PORTABLE || PORTABLE40) && !ANDROID && !NET_CORE
+#if (DOTNET || PORTABLE || PORTABLE40) && !ANDROID && !NET_CORE && !WIN_UI
```

Comment:
What is `WIN_UI` ? It is not defined anywhere?

Reply:
I believe it is defined for the supported platform. @mikeparker104 please confirm

Reply:
Can you please check yourself @SameerK-MSFT ? Visual Studio is pretty good at this, you just add a bit of code like: 

![image](https://user-images.githubusercontent.com/12273384/177298711-4554cc43-de71-44f5-8eaa-78e7debb159e.png)

then switch the TFM drop-down to various settings and see if the `code` is grayed out or not.

![image](https://user-images.githubusercontent.com/12273384/177298794-a8349504-fba6-4083-8c17-b3193ce76928.png)


Even if this is defined, I'd rather not use it, because it is obscure. Please use an existing constant or add this to the list of pre-compilation constants.

Reply:
That constant has no effect. Removed.